virt/virt-2-5-release-notes.adoc

```diff
@@ -64,6 +64,8 @@ Other operating system templates shipped with {VirtProductName} are not supporte
 //Placeholder for new content.
 
 //CNV-7170 - CDI import of container disks is faster and more efficient
+* The Containerized Data Importer (CDI) can now import containerDisk storage volumes from the container image registry at a faster speed and allocate storage capacity more efficiently. CDI can pull a containerDisk image from the registry in about the same amount of time that it would
+take for an HTTP import. You can import the disk into a PersistentVolumeClaim (PVC) equal in size to the disk image to use the underlying storage more efficiently.
```

Comment:
Maybe "...same amount of time as it would to import from an HTTP endpoint."
Also we're no longer supposed to hard-wrap.